src/locales/tr.json

```diff
@@ -231,6 +231,38 @@
     "aboutUsHeader": "Hakkımızda",
     "aboutUsText": "Teknolojiyi insanların yararına kullanmakta ortak paydasını bulan çok uluslu bir gençlik grubuyuz. Orta Doğu ve Kuzey Afrika'daki çocukları eğitici oyunlar aracılığıyla çevre sorunları ve iklim değişikliği konusunda eğitmek için birlikte çalışan 5 genç geliştirici ve 1 baş mühendisten oluşan bir grup. Hedef kitlemiz MENA'da yaşayan 8-13 yaş arası çocuklar; bu nedenle, web sitemizi çocukların ve öğretmenlerinin / ebeveynlerinin web sitesinde kolayca gezinmelerine yardımcı olmak için üç ana dilde sunuyoruz.",
     "whyWeDoItHeader": "Neden yapıyoruz",
-    "whyWeDoItText": "İnsanların doğayı ve özelliklerini korumak gibi ahlaki bir yükümlülüğü vardır, çünkü çevreyi korumak insanlığı toksik maddelerden ve gıda kalitemizi korur. MENA'daki yeni neslin, yerel ve küresel ölçekte insani ve sosyal konulara inanmak ve fayda sağlamak için hareket etmek için doğru araçlarla desteklenmesi gerektiğine inanıyoruz. Doğru araçlar verildiğinde, bir nesil aktivist yetiştirmek imkansız değildir."
+    "whyWeDoItText": "İnsanların doğayı ve özelliklerini korumak gibi ahlaki bir yükümlülüğü vardır, çünkü çevreyi korumak insanlığı toksik maddelerden ve gıda kalitemizi korur. MENA'daki yeni neslin, yerel ve küresel ölçekte insani ve sosyal konulara inanmak ve fayda sağlamak için hareket etmek için doğru araçlarla desteklenmesi gerektiğine inanıyoruz. Doğru araçlar verildiğinde, bir nesil aktivist yetiştirmek imkansız değildir.",
+    "teamInfo": {
+      "ammar": {
+        "name": "Ammar Sammour",
+        "bio": "",
+        "role": "Ekip Lideri"
+      },
+      "ibrahim": {
+        "name": "Ibrahim Aljarad",
+        "bio": "Hey, ben Suriye'den İbrahim, Flatiron kodlama okulunda kodlama okudum ve Alfurat üniversitesinde ekonomi okudum.Re:coded organizasyonla bir ön uç web geliştirme temel eğitim programına katıldım ve oradan mezun oldum. Kodlama dışındaki diğer deneyimlerim, saha yapım ekiplerine teknik destek sağlayan uydu yapımcısı olarak çeşitli kanallar ve ajanslarla çalıştım.",
+        "role": "Web Arayüz Geliştirici"
+      },
+      "majd": {
+        "name": "Majd Ajroudi",
+        "bio": "Çok yönlü becerilere, çeşitli mühendislik alanlarında deneyime ve sosyal sorunları çözmek için teknolojiyi kullanma konusunda istekli bir genç geliştirici.",
+        "role": "Web Arayüz Geliştirici"
+      },
+      "sule": {
+        "name": "Sule Sinikoglu",
+        "bio": "Küresel Zorluklar üzerinde olumlu etki yaratmak için 'iyi kod' yazan bir Web Geliştiricisi.",
+        "role": "Web Arayüz Geliştirici"
+      },
+      "tarkan": {
+        "name": "Tarkan Mansuroglu",
+        "bio": "Re:coded Ön Uç Web Geliştirme Eğitim Kampı eğitimli ve kısmen kendi kendine öğretilen bir ön uç web geliştiricisi. Aynı zamanda tutkulu bir öğrenci, bilgisayar meraklısı, oyun tutkunu ve spor tutkunu. Becerilerini test etmek ve geliştirmek için gerçek hayattaki sorunları çözmeyi seviyor. Onu sosyal medyadan takip edin!",
```

Comment:
You may update like
"Re:coded Web Arayüz Geliştirme eğitim programı mezunu ve kısmen kendi kendine öğrenen bir web arayüz geliştiricisi. Ayrıca tutkusu öğrenmek olan, bilgisayarlar konusunda meraklı, bilgisayar oyunlarına meraklı bir spor tutkunu. Becerilerini test etmeyi ve geliştirmek için gerçek hayata dair sorunları çözmeyi seviyor. Onu sosyal medyadan takip edin!"

Reply:
PTAL